c/zss.c

```diff
@@ -1215,6 +1218,9 @@ int main(int argc, char **argv){
       installUnixFileTouchService(server);
       installUnixFileMetadataService(server);
       installUnixFileChangeOwnerService(server);
+#ifdef __ZOWE_OS_ZOS
```

Comment:
Why only ZOS?

Reply:
`chtag` and tagging is an ability that only exists on ZOS.